packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -45,6 +46,21 @@ function buildMenuLabel( title, id, status ) {
 // Save a boolean to prevent us creating a fallback more than once per session.
 let hasCreatedFallback = false;
 
+function DelayedSpinner( { delay = 250, ...props } ) {
```

Comment:
@scruffian I've experimented with this component which won't render the spinner until minimum of 250ms has elapsed.